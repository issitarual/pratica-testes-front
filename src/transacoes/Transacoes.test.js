import React from 'react';
import { render } from '@testing-library/react';
import Transacoes from './Transacoes';

describe('Componente de transação do extrato', () => {
    it('O snapshot do componente deve permanecer sempre o mesmo', () => {
        const { container } = render(
            <Transacoes
                transacoes={[
                    {valor: 10, transacao: "saque", data: "10/08/2020", id: 1},
                    {transacao: "deposito", valor: 20, data: "26/09/2020", id: 2}
                ]}
            />
        )

        expect(container.firstChild).toMatchSnapshot();
    })
})