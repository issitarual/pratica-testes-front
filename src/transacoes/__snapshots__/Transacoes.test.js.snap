// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transação do extrato O snapshot do componente deve permanecer sempre o mesmo 1`] = `
<div
  data-testid="transacoes"
>
  <div
    class="transacao-container"
  >
    <p>
      10/08/2020
    </p>
    <p>
      saque
    </p>
    <p>
      R$ 
      10
    </p>
  </div>
  <div
    class="transacao-container"
  >
    <p>
      26/09/2020
    </p>
    <p>
      deposito
    </p>
    <p>
      R$ 
      20
    </p>
  </div>
</div>
`;
